airflow/airflow.go

```diff
@@ -115,6 +115,7 @@ func Init(path, airflowImageName, airflowImageTag string) error {
 		"README.md":                            Readme,
 		"tests/dags/test_dag_example.py":       DagExampleTest,
 		".astro/test_dag_integrity_default.py": DagIntegrityTestDefault,
+		".astro/dag_integrity_exceptions.txt":  "# Add dag files to exempt from parse test below. ex: dags/<test-file>",
```

Comment:
How would users with existing projects fix this? Do we need something in our release notes for this ?

Reply:
yeah If they want this file they would need to delete `.astro/test_dag_integrity_default.py` and run `astro dev init`